### DISCUSSION TITLE
为啥子 ColorsDark.xaml 的 BackgroundColor 是 #eeeeee

### DISCUSSION THREAD

**Discussion by choking-chips** [HandyOrg/HandyControl#1090] [General]
第 33 行

**Comment by Yopler** [comment]
Hi 🖐,

Actually the use of the `BackgroundColor `is very specific in Handy Control. One of his only use is for the background of the Progress bar. 
The main colors use for backgrounds in Handy are the `RegionBrush`, `SecondaryRegionBrush `and `ThirdlyRegionBrush`

**Reply by choking-chips** [reply]
Got it, thanks!
